doc/usecases/elektra_web/UC_add_cluster.md

```diff
@@ -0,0 +1,20 @@
+# Use Case: Add cluster
+
+## Summary
+
+Title: Add cluster
+Scope: Setup
+Level: User Goal
+Actors: User (usually a sysadmin)
+Brief: User groups multiple instances into a cluster.
+
+## Scenarios
+
+Precondition: Add instance (multiple times)
+Main success scenario: User successfully creates a cluster of multiple instances.
```

Comment:
They could be interesting to:

1. define multiple groups if you use a cluster per application (all mail, web servers ...)
2. for failover (two clusterd for endpoints refer to other clusters, if one machine is gone, the system is still reachable)

Also only connecting to elektrad is essential, nodejs is a too heavy dependency for many systems.